controllers/config/defaults.go

```diff
@@ -191,6 +195,14 @@ func createResourceRequirement(RequestsCPU resource.Quantity, RequestsMemory res
 	}
 }
 
+func GetStringConfigOrError(configName string) (string, error) {
```

Comment:
Optional: The `OrError` seems unnecessary. It's implied that it returns an error based on the return value types.